src/main/scala/tech/cryptonomic/conseil/tezos/michelson/renderer/MichelsonRenderer.scala

```diff
@@ -9,14 +9,14 @@ object MichelsonRenderer {
     def render(): String = self match {
 
       // instructions
-      case MichelsonSingleInstruction(prim, List()) => prim
-      case MichelsonSingleInstruction(prim, args) => s"$prim ${args.map(_.render()).mkString(" ")}"
+      case MichelsonSingleInstruction(prim, List(), List()) => prim
```

Comment:
Just catched this, here you can and will probably want to pattern match against `Nil` which is a case object.

Otherwise we call a method `apply` (or `unapply` not sure) on the `List` object just create the equivalent value.